### ISSUE TITLE
Support org-agenda-show-future-repeats to avoid cluttering the agenda with repeated tasks

### ISSUE THREAD

**Issue by Maltimore** [nvim-orgmode/orgmode#484]
### Does this feature exist in Emacs orgmode core?

Yes

### Orgmode link

There doesn't seem to be an entry for it in the orgmode manual, but it's discussed in various places on the internet, including here:
https://emacs.stackexchange.com/questions/12609/org-mode-prevent-future-repetitive-entries-from-showing-up-in-agenda-view

### Feature value

I have a few tasks that I want to do every day. So I have repeater tasks with date `.+1d`. However, these tasks now clutter my agenda. I only want to see them for the given day. Emacs Org provides a variable to do this (see above).

**Comment by jgollenz** [comment]
For quick reference, the docstring for `org-agenda-show-future-repeats` says

```
Non-nil shows repeated entries in the future part of the agenda.
When set to the symbol 'next' only the first future repeat is shown
```

and for `org-agenda-prefer-last-repeat`
```
Non-nil sets date for repeated entries to their last repeat.

When nil, display SCHEDULED and DEADLINE dates at their base
date, and in today’s agenda, as a reminder.  Display plain
time-stamps, on the other hand, at every repeat date in the past
in addition to the base date.

When non-nil, show a repeated entry at its latest repeat date,
possibly being today even if it wasn’t marked as done.  This
setting is useful if you do not always mark repeated entries as
done and, yet, consider that reaching repeat date starts the task
anew.

When set to a list of strings, prefer last repeats only for
entries with these TODO keywords.
```

**Comment by edpyt** [comment]
Have you found a way to implement a feature similar to this one? Or do we keep waiting for the feature to be implemented?

**Comment by Maltimore** [comment]
No, I haven't found a way to implement it. Personally, I figured out that having daily tasks in my agenda didn't help either way, as I developed "blindness" for them after a few days. But everyone's different, I think it could still make sense to implement this feature eventually. Probably it makes a lot of sense for weekly or monthly tasks.
